packages/react-components/src/theming/themes.ts

```diff
@@ -19,12 +19,37 @@ export interface CallingTheme {
   };
 }
 
+/**
+ * Custom Fluent theme palette used by chat related components in this library.
+ *
+ * @public
+ */
+export interface ChatTheme {
```

Comment:
Should this interface be @beta and then the conditional compile changes applied to the theme object instead?